Packs/Base/Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.py

```diff
@@ -90,7 +90,7 @@ def predict_phishing_words(model_name, model_store_type, email_subject, email_bo
     explain_result_hr = dict()
     explain_result_hr['TextTokensHighlighted'] = highlighted_text_markdown
     explain_result_hr['Label'] = predicted_label
-    explain_result_hr['Probability'] = "%.2f" % predicted_prob
+    explain_result_hr['Confidence'] = "%.2f" % predicted_prob
```

Comment:
please add another field called confidence and don't replace the current one so we won't have backward compatabilites issues. Also you need to update the outputs - add new output (also have this, the old one)